### PR TITLE
Enhance desktop app

### PR DESCRIPTION
**Pull request by JohannesMerkt**
This is what a came up with for https://github.com/petrvecera/coh2ladders/issues/140 
grafik
There might be a better way to do this.

https://github.com/petrvecera/coh2ladders/issues/138 done
grafik

Also:
- different view when in a game and when waiting for a game
- created a splash screen animation for installer
- standadized naming of components.
- reduced RAM usage

### CODE REVIEW

**Comment by petrvecera** [comment]
This looks  awesome! 👍 The gif is great.
image
From the UX perspective I would probably improve this little bit.
1. "Win rate for team composition"
I feel like some people might take this wrongly they might think that this is winrate for the particular team players - from my experience some people always thinks incorrectly even when you specify it well ... I would make it so it's more clear that it's just about the factions. 
What about "Winrate for this team faction composition"  - so it's clear that it's about the factions.

2. The sizing doesn't feel 100% right, I would try to decrease the sizing of the icons little bit - I feel like they are too big in compare to the text and other elements 
image


Otherwise looks great

**Comment by JohannesMerkt** [comment]
Thank you for your feedback. I will make these improvements as soon as I have time. 

The total win ratio is a bit controversial. It is my attempt to summarize the 3 metrics average level, win ratio and map win ratio. Its probably easer if I just show you the code for the calculation: 
grafik

**Comment by petrvecera** [comment]
I see thanks for explaining.  We need to name it differently, I would call it "Victory Chance Probability" and I would put question icon behind it with tooltip - we can use this https://github.com/petrvecera/coh2ladders/blob/master/packages/web/src/components/helper.tsx

And there I would write:
"This is probability of victory based on average team level, average players winrate and team factions matchup winrate" < something like that, maybe throw in the formula? 

We can also put the question icon with tooltip to the map winrate and in that tooltip we can heave "Based on coh2stats.com map analysis." and the map analysis can be hyperlink to the web with the analysis. Like this https://coh2stats.com/map-stats?range=month&type=4v4&map=8p_redball_express&timeStamp=1643673600

File: packages/app/src/main/streamerOverlay/centerView.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Col, Row } from "antd";
+import Col from "antd/lib/grid/col";
```
[the remark this answers is not in the document]
Comment (JohannesMerkt) [reply packages/app/src/main/streamerOverlay/centerView.tsx:1]: Good point! Guess this wasnt necessary 😄

**Review by petrvecera — APPROVED** [review]

**Pull request merged**